sdk/core/azure-core/inc/azure/core/http/http.hpp

```diff
@@ -473,12 +490,19 @@ namespace Azure { namespace Core { namespace Http {
     }
 
     /**
-     * @brief Add an HTTP header.
+     * @brief Adds an HTTP header to the Request.
```

Comment:
I wrote these comments as "Add", "Make", "Create", "Delete" etc. It is uniform across all comments.
I think "Add" is better than "Adds": imagine navigating intellisense, and it tells you the action. When we write "Adds" it is like speaking in 3rd person form.

Reply:
also, write it as `@Request`

Reply:
done

Reply:
> also, write it as `@Request`

@antkmsft - how come?